src/com/Jacksonnn/DCCore/GeneralMethods.java

```diff
@@ -12,15 +12,50 @@
 
 import java.awt.*;
 import java.util.Collection;
+import java.util.HashMap;
 import java.util.Objects;
+import java.util.UUID;
 
 public class GeneralMethods {
+	private static DCCore plugin;
+
+	public GeneralMethods(DCCore plugin) {
+		GeneralMethods.plugin = plugin;
+	}
+
 	public static String prefix = ChatColor.DARK_GRAY + "[" + ChatColor.GRAY + ChatColor.BOLD + "DarkcrestMC" + ChatColor.DARK_GRAY + "]" + ChatColor.YELLOW + " ";
 	public static String errorColor = prefix + ChatColor.DARK_RED + "Error! " + ChatColor.RED;
 	public static String successColor = prefix + ChatColor.GREEN + "";
 	public static String disableColor = prefix + ChatColor.RED + "";
 	public static String serverPrefix = ChatColor.DARK_GRAY + "[" + ChatColor.GRAY + ChatColor.BOLD + "DarkcrestMC" + ChatColor.DARK_GRAY + "]" + ChatColor.YELLOW + " ";
 	private static int staffNotification = 0;
+	private static HashMap<UUID, DCPlayer> dcPlayers = new HashMap<>();
+
+	public static HashMap<UUID, DCPlayer> getAllDCPlayers() {
+		return dcPlayers;
+	}
+
+	public static DCPlayer getDCPlayer(UUID uuid) {
+		if (dcPlayers.containsKey(uuid)) {
+			return dcPlayers.get(uuid);
+		} else {
+			return null;
+		}
+	}
+
+	//FROM DATABASE
+	public static void addDCPlayer(DCPlayer dcPlayer) {
+		dcPlayers.put(dcPlayer.getUuid(), dcPlayer);
+	}
+
+	//NEW DCPLAYER
+	public static void createDCPlayer(Player player) {
+		dcPlayers.put(player.getUniqueId(), new DCPlayer(player, plugin.getDCPM()));
+	}
+
+	public static void removeDCPlayer(DCPlayer dcPlayer) {
+		dcPlayers.remove(dcPlayer.getUuid());
+	}
 
 	public enum Elements {
 		AIR,
```

Comment:
Perhaps insert Spirit into the Elements enum

Reply:
oh that needs to be removed— it was from a part of DCCore that I removed and made into a separate plugin.